deploy/charts/alluxio/templates/jobs/load-script.yaml

```diff
@@ -0,0 +1,66 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+{{- $name := include "alluxio.name" . }}
+{{- $fullName := include "alluxio.fullname" . }}
+{{- $chart := include "alluxio.chart" . }}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $fullName }}-alluxio-load
+  labels:
+    name: {{ $fullName }}-alluxio-load
+    app: {{ $name }}
+    chart: {{ $chart }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  load.go: |-
+    package main
+
+    import (
+        "context"
+        "fmt"
+        "os"
+        "os/exec"
+        "strings"
+        "time"
+    )
+
+    func main() {
+        if len(os.Args) != 2 {
+            panic("Load command only takes one argument")
+        }
+        path := os.Args[1]
+        cmd := exec.Command(fmt.Sprintf("/opt/alluxio/bin/alluxio fs load %s --progress", path))
+        if _, err := cmd.Output(); err != nil {
+            panic(err)
+        }
+        waitUntilLoadFinish(path)
+    }
+
+    func waitUntilLoadFinish(path string) {
+        status := make(chan int, 1)
+        ctx, cancel := context.WithTimeout(context.Background(), 10 * time.Second)
+        defer cancel()
+        go func() {
+            for range time.Tick(15 * time.Second) {
+                output, _ := exec.CommandContext(ctx, "/opt/alluxio/bin/alluxio fs load %s --progress", path).Output()
```

Comment:
why ignore the error instead of checking and sending a status 1 if error?

Reply:
Sending status 1 will cause the script to exit. The script is the only process for the container, so if it exits, k8s will regard this job has finished. But the alluxio load job may not have finished yet. So we shall only exit the script after the alluxio load job successfully finishes or fails.

Reply:
actually can we log both the output and error? if there's an error, the user needs to see it to react to it.
but even if there's no error, we'd be stuck in an infinite loop if the output string changes and it never returns either of the expected substrings. it'll also appear like an infinite loop with no information on what's happening

Reply:
If there's a way to show "the progress command is always failing and we are in an infinite loop rn" that would be even better, instead of user has to look at the log to figure that out. K8s job doesn't have a way to show this though.